lib/routes/crossbell/notes/character.ts

```diff
@@ -1,6 +1,5 @@
 import { Route } from '@/types';
 import got from '@/utils/got';
-import { getItem } from './utils';
```

Comment:
You don't need to change this file at all.
All you need to fix the export of `utils` to something like:
```diff
-export default {
-    getItem: (note) => {
+export const getItem = (note) => {
```

Reply:
Thanks for the reply. I think the above format may be better for character RSS output.
can I modify that format to `utils`.

Reply:
Or which format is more suitable for character. For example, name should be handle instead of id, so response_getCharacter() is needed.

Reply:
The current format of `getItem` is already fine.

Reply:
Okay, do I need to change character getItem to utils' ?

Reply:
> The current format of `getItem` is already fine.

I have modified it as expected. Thanks for the correction. You did a great job😘